### PR TITLE
release: Create github release after create tag.

### PR DESCRIPTION
**Pull request by jcvenegas**
After create tags create a github release.

After create the github release this can be edited to
add new assets or modify release notes.

### CODE REVIEW

File: release/tag_repos.sh [hunk 1]
```diff
@@ -103,16 +105,31 @@ tag_repos() {
...
+create_github_release(){
+	repo_dir=${1:-}
+	tag=${2:-}
+	[  -d "${repo_dir}" ] || die "No repository directory"
+	[  -n "${tag}" ] || die "No repository directory"
+	if ! "${hub_bin}" release | grep "${tag}"; then
+		info "Creating Github release"
+		"${hub_bin }" -C "${repo_dir}" release create -m "${PROJECT} ${tag}" "${tag}"
+	else
+		info "Github relase already created"
```
Comment (jodh-intel) [inline release/tag_repos.sh]: Nit: typo: "release".

**Comment by jodh-intel** [comment]
Hi @jcvenegas - please rebase to resolve conflict.

**Comment by jcvenegas** [comment]
@jodh-intel thx, branch updated, waiting for CI to merge.

**Comment by jodh-intel** [comment]
Thanks @jcvenegas. Still...

lgtm.

Ping @kata-containers/packaging y'all!

**Comment by devimc** [comment]
lgtm

**Pull request merged**